.swift-format

```diff
@@ -47,7 +47,7 @@
     "UseLetInEveryBoundCaseVariable" : false,
     "UseShorthandTypeNames" : true,
     "UseSingleLinePropertyGetter" : false,
-    "UseSynthesizedInitializer" : true,
+    "UseSynthesizedInitializer" : false,
```

Comment:
Needed to turn this off, as it wanted me to remove a "default" struct initializer, but it had an inlining attribute which was important.

Reply:
Can we report this upstream?

Reply:
Filed an issue on us to re-enable when swift-format 5.9 is out: https://github.com/apple/swift-openapi-generator/issues/250